.doc_gen/templates/zonbook/service_chapter_template.xml

```diff
@@ -26,6 +29,27 @@
     </info>
     <para>The following code examples show how to use {{.ServiceEntity.Short}} with an &AWS; software development kit (SDK).
         {{.ServiceEntity.Caveat}}</para>
+    <variablelist role="termdef">
+        <para>The examples are divided into the following categories:</para>
+        {{- if $actions.Examples}}
+        <varlistentry>
+            <term>Actions</term>
+            <listitem><para>Excerpts that illustrate how to write code to call individual service functions.</para></listitem>
+        </varlistentry>
+        {{- end}}
+        {{- if $scenarios.Examples}}
+        <varlistentry>
+            <term>Scenarios</term>
+            <listitem><para>Examples that demonstrate ways you can write code to accomplish a specific task with a single service.</para></listitem>
+        </varlistentry>
+        {{- end }}
+        {{- if $cross_service.Examples}}
+        <varlistentry>
+            <term>Cross-service</term>
+            <listitem><para>Sample applications that combine multiple &AWS; services.</para></listitem>
```

Comment:
<listitem><para>Sample applications that work across multiple &AWS; services.</para></listitem>

Reply:
Done